src/EnergyPlus/api/runtime.cc

```diff
@@ -54,7 +54,7 @@
 #include <EnergyPlus/UtilityRoutines.hh>
 
 void cClearAllStates() {
-    EnergyPlusData state;   //THIS IS TEMPORARY
+    EnergyPlus::EnergyPlusData state;   //THIS IS TEMPORARY
```

Comment:
This definitely needs to be temporary.  I will do some thinking about this.  I think the main point is that now when someone wants to run an EnergyPlus simulation as a library, they need to own the state.